OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -52,7 +52,7 @@ class NukePowerInfo : SupportPowerInfo, IRulesetLoaded, Requires<BodyOrientation
 		public readonly string TrailImage = null;
 
 		[Desc("Loop a randomly chosen sequence of TrailImage from this list while this projectile is moving.")]
-		[SequenceReference("TrailImage")] public readonly string[] TrailSequences = { "idle" };
+		[SequenceReference("TrailImage")] public readonly string[] TrailSequences = { };
```

Comment:
NukeLaunch assumes that this is not empty if `TrailImage` is defined.
Can you please add a `TraitSequences.Any()` check there or throw an exception from `RulesetLoaded` below?

Reply:
Done.